src/pages/home-page/Carousel.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { Pagination } from 'swiper/modules';
+import 'swiper/css';
+import 'swiper/css/pagination';
+import Card from "../../Card";
+import {FC} from 'react';
+import useMobileCheck from "../../shared/hooks/useMobileCheck";
+import arrowsIcon from "../../assets/icons/arrows.svg";
+import './Carousel.scss'
+
+
+interface CarouselProps {
```

Comment:
please move to directory : types/interfaces
and put this interface in separate file